css/main.css

```diff
@@ -96,3 +102,81 @@ header .search-area {
 header .search-area ul li {
   display: inline-block;
 }
+/*
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+*/
+
+#top-area {
+  padding-top: 200px;
+  padding-right: 15px;
+  padding-bottom: 90px;
+  padding-left: 15px;
+
+  background-image: linear-gradient(
+      48deg,
+      rgba(0, 0, 0, 1) 0%,
+      rgba(0, 0, 0, 0.6839110644257703) 31%,
+      rgba(7, 119, 189, 0.5354516806722689) 100%
+    ),
+    url('../images/cover-photo/cover-photo.png');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+#top-area .profile-info {
+  display: flex;
+}
+
+#top-area .profile-info .user-image {
+  padding: 10px;
+  display: inline-block;
+}
+
+#top-area .profile-info .user-image img {
+  border-radius: 50%;
+  /* filter: grayscale(100%); */
```

Comment:
remove this commented code

Reply:
Thanks for pointing that out. Let me do that.

Reply:
Done